.golangci.yml

```diff
@@ -68,6 +68,11 @@ linters-settings:
     min-occurrences: 3
   depguard:
     list-type: blacklist
+    rules:
+      main:
+        deny:
+          - pkg: "io/ioutil"
```

Comment:
usually you can replace the calls to `ioutil` to the matching calls either in `os.` or `io.` packages